scripts/upgrade

```diff
@@ -76,7 +76,12 @@ ynh_install_php7
 
 # Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source "$final_path"
-
+init_composer_update $final_path
+(
+cd $final_path && sudo /usr/bin/php7.2  bin/console cache:clear --env=prod
+cd $final_path && sudo /usr/bin/php7.2  bin/console cache:warmup --env=prod
+cd $final_path && sudo /usr/bin/php7.2 bin/console doctrine:migrations:migrate
```

Comment:
Pleas add the `-n` flag, otherwise the user would need to hit enter here as well